packages/cloud_firestore/cloud_firestore_platform_interface/pubspec.yaml

```
@@ -4,15 +4,15 @@ version: 3.0.1
 homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/cloud_firestore/cloud_firestore_platform_interface
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
+  sdk: ">=2.12.0-133.7.beta <3.0.0"
```

```suggestion
  sdk: ">=2.12.0-0 <3.0.0"
```

This is fine as it was